scripts/exglobal_atmos_tropcy_qc_reloc.sh

```diff
@@ -11,157 +11,154 @@
 ############################################################################
 
 # Make sure we are in the $DATA directory
-cd "${DATA}"
+cd "${DATA}" || exit
 
 tmhr=${tmmark:2:2}
 cdate10=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${tmhr} hours")
 
 NET_uc=${RUN^^}
 tmmark_uc=${tmmark^^}
 
-iflag=0
+export iflag=0
```

Comment:
No need to export this.  `iflag` isn't used upstream.
```suggestion
iflag=0
```

Reply:
If it's not being used downstream, then it isn't being used at all. Removing.